binary_parser.go

```diff
@@ -144,21 +142,22 @@ func (bp *binaryParser) parseSingleton(marker byte) (*plistValue, error) {
 
 func (bp *binaryParser) parseInteger(marker byte) (*plistValue, error) {
 	nbytes := 1 << (marker & 0xf)
-	// The binary plist format allows for 16 byte integers, but we don't have
-	// any easy way to decode them (have to use math/big package?)
-	if nbytes > 8 {
-		return nil, fmt.Errorf("can't decode integer longer than 8 bytes (%d)", nbytes)
-	}
-	// Create an 8-byte buffer to read into.  If nbytes < 8, then we
-	// only read into the trailing bytes and zero-fill the rest.
-	buf := make([]byte, 8)
-	_, err := bp.Read(buf[8-nbytes:])
+	if nbytes > 16 {
+		return nil, fmt.Errorf("cannot decode integer longer than 16 bytes (%d)", nbytes)
```

Comment:
add `plist: ` in front of the error, so that a log would read
`plist: cannot decode integer longer than 16 ...`